lib/get-category-path.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+var path = require('path'),
+  utility = require('./utility');
+
+
+/**
+ * Determines the category directory structure from the pattern's data
+ *
+ * @param {Object} patternObject  individual pattern's data (from /pattern-name/pattern.yml)
+ * @param {Object} options  pattern-importer options
+ *
+ * @return {String} categoryPath  internal path url to category
+ *
+ * @requires ./lib/utility
+ * @requires ./lib/category-name-converter
+ */
+exports.getCategoryPath = function (patternObject, options) {
```

Comment:
@scottnath @ericduran
cc: @conortm @beynya @gvorbeck @nikkiana @rebmullin @sergesemashko
It is drastic in that many hundreds, maybe thousands of lines of code would need to be changed just to adhere to a coding standard. There is nothing all that "mistaken" or "proper" about any of the popular standards out there. They were just drawn up to be consistent. And they only really matter if someone else sees your code.

That said, I think we should go with the Drupal JS coding standards since they're extremely well-defined, have linter rc files already drawn up, and are already well-adopted by much of NBCU. Thoughts?